charts/github-actions-runner-operator/Chart.yaml

```diff
@@ -30,11 +30,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.3.0
+version: 2.3.2
```

Comment:
```suggestion
version: 2.3.1
```

Reply:
hey, I'm well aware of this. of course it's just that I have actually made two releases and published in my repo)